UIKitTemplate/UIKitTemplate/AchorViewController.swift

```diff
@@ -0,0 +1,95 @@
+// AchorViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Экран с якорями
+final class AchorViewController: UIViewController {
```

Comment:
```suggestion
final class AnchorViewController: UIViewController {
```
старайся внимательнее писать, потом исправить код (особенно на работе) в разы тяжелее чем сразу))) 
у меня тоже с внимательностью проблемы, но стараюсь проверять по 10 раз всё поэтому